docs/get-started/tutorials/pulsar.md

```diff
@@ -0,0 +1,207 @@
+
+## Prerequisites
+
+You will need the following to complete this tutorial:
+- A local running [Quickwit instance](/docs/get-started/installation)
+- A local running [Pulsar instance](https://pulsar.apache.org/docs/next/getting-started-standalone/). 
+
+### Quickwit setup
+
+[Download](/docs/get-started/installation) Quicwkit and start a server. Then open a new terminal to execute CLI commands with the same binary. 
+
+```bash
+./quickwit run
+```
+
+Test the cluster is running:
+
+```bash
+./quickwit index list
+```
+
+### Pulsar setup
+
+```
+wget https://archive.apache.org/dist/pulsar/pulsar-2.11.0/apache-pulsar-2.11.0-bin.tar.gz
+tar xvfz apache-pulsar-2.11.0-bin.tar.gz
+cd apache-pulsar-2.11.0
+bin/pulsar standalone
+```
+
+## Prepare Quickwit
+
+First, let's create a new index. Here is the index config and doc mapping corresponding to the schema of stackoverflow posts:
+
+```yaml title="index-config.yaml"
+#
+# Index config file for stackoverflow dataset.
+#
+version: 0.4
+
+index_id: stackoverflow
+
+doc_mapping:
+  field_mappings:
+    - name: user
+      type: text
+      fast: true
+      tokenizer: raw
+    - name: tags
+      type: array<text>
+      fast: true
+      tokenizer: raw
+    - name: type
+      type: text
+      fast: true
+      tokenizer: raw
+    - name: title
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: body
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: questionId
+      type: u64
+    - name: answerId
+      type: u64
+    - name: acceptedAnswerId
+      type: u64
+    - name: creationDate
+      type: datetime
+      fast: true
+      input_formats:
+        - rfc3339
+      precision: seconds
+  timestamp_field: creationDate
+
+search_settings:
+  default_search_fields: [title, body]
+
+indexing_settings:
+  commit_timeout_secs: 10
+```
+
+Execute these Bash commands to download the index config and create the `stackoverflow` index.
+
+```bash
+# Download stackoverflow index config.
+wget -O stackoverflow.yaml https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/stackoverflow/index-config.yaml
+
+# Create index.
+./quickwit index create --index-config stackoverflow.yaml
+```
+
+## Create and populate a Pulsar topic
+
+Now, let's create a Pulsar topic and send some messages into it.
+
+```bash
+# Execute these commands in the pulsar directory.
+bin/pulsar-admin tenants create quickwit
+# Check it worked.
+bin/pulsar-admin tenants list
+bin/pulsar-admin namespaces create quickwit/pulsar
+# Set retention policy for the namespace
+bin/pulsar-admin namespaces set-retention quickwit/pulsar --time 10m --size 1G
+# Create topic.
+bin/pulsar-admin topics create-partitioned-topic quickwit/pulsar/stackoverflow -p 4
+# Check it worked.
+bin/pulsar-admin topics list-partitioned-topics quickwit/pulsar
```

Comment:
I'm tempted to say we should remove this for now for the following reasons:

- We can use the default namespace for this tutorial to get a basic pulsar setup working
- Partitioned topics are not needed to get started although it doesn't do any harm having them here, for simplicity though it's easier to just let pulsar create it for us when we push docs to it.